### ISSUE TITLE
Should align text to the right

### ISSUE THREAD

**Issue by unitehenry** [striven-erp/striven-editor#118]

**Comment by XIZT07** [comment]
Can i take up this issue?

**Comment by unitehenry** [comment]
@XIZT07 Absolutely! You can write this test here: ```striven-editor/demo/src/test.js```

**Please reach out for any help.**

**Comment by XIZT07** [comment]
Please provide a clear description - if any content has to be aligned right or tests are to be written for aligning right?
I didn't get it clearly!!

**Comment by unitehenry** [comment]
I think the test should do the following:

1. Set some content in the editor with ```se.setContent```
2. Use ```se.executeCommand``` to apply align right formatting
3. Should call ```se.getContent``` and check to make sure the HTML has right alignment on the content that was formatted.
